v3/src/components/graph/models/graph-data-configuration-model.ts

```diff
@@ -397,27 +397,44 @@ export const GraphDataConfigurationModel = DataConfigurationModel
         }
       })
     },
-    binDetails() {
+    binDetails(_binAlignment?: number, _binWidth?: number, reset = false) {
```

Comment:
It doesn't appear that `reset` is ever used, in which case it can be removed.

Reply:
Yes, I forgot to get rid of that. Thanks for pointing it out. I've removed it in the draggable bin boundaries branch I'm working on.